cli/tools/coverage.rs

```diff
@@ -37,31 +37,76 @@ impl CoverageCollector {
     Self { dir, session }
   }
 
-  pub async fn start_collecting(&mut self) -> Result<(), AnyError> {
+  async fn enable_debugger(&mut self) -> Result<(), AnyError> {
     self.session.post_message("Debugger.enable", None).await?;
+
+    Ok(())
+  }
+
+  async fn enable_profiler(&mut self) -> Result<(), AnyError> {
     self.session.post_message("Profiler.enable", None).await?;
-    self
+
+    Ok(())
+  }
+
+  async fn disable_debugger(&mut self) -> Result<(), AnyError> {
+    self.session.post_message("Debugger.disable", None).await?;
+
+    Ok(())
+  }
+
+  async fn disable_profiler(&mut self) -> Result<(), AnyError> {
+    self.session.post_message("Profiler.disable", None).await?;
+
+    Ok(())
+  }
+
+  async fn start_precise_coverage(
+    &mut self,
+    parameters: StartPreciseCoverageParameters,
+  ) -> Result<StartPreciseCoverageReturnObject, AnyError> {
```

Comment:
Could be cut this down quite a bit if parameters and return object were generalised on `self.post_message` and the stubs only helped with type inference.